### ISSUE TITLE
Problem installing gewombat from git clone

### ISSUE THREAD

**Issue by felipealencar** [jgrss/geowombat#246]
Hello,

I'm trying to install gewombat, but after trying in my local environment and even on Google Colab, the same error occurs:

` Running command git clone -q https://github.com/jgrss/geowombat '...\AppData\Local\Temp\pip-req-build-ibye76t_'
ERROR: Some build dependencies for git+https://github.com/jgrss/geowombat conflict with the backend dependencies: cython==3.0.0b1 is incompatible with cython>=0.29.0.`

![image](https://user-images.githubusercontent.com/3163733/222564840-f1775207-6df8-4134-8ae3-7956bf05f0ae.png)

I also tried the Conda installation, but it is taking too long, and Idk if it's feasible to do that on Google Colab.

Any idea about what is going on?

Thanks.

**Comment by jgrss** [comment]
Thanks for raising this @felipealencar. It looks like we need to pin cython to <3.0 as 3.0 alpha and beta releases appear not to be backwards compatible. I'll test this out and get a PR for this issue.

**Comment by mmann1123** [comment]
In the meantime you can run this before you do the geowombat install

pip install -U --no-cache-dir cython>=0.29.*

**Comment by jgrss** [comment]
To follow on @mmann1123's comment, you might want to uninstall cython prior to installing geowombat.
```commandline
pip uninstall cython -y
pip install geowombat@git+https://github.com/jgrss/geowombat
```

That will probably install `Cython>=3.x`. My guess is that you already had `Cython<3.x` installed, geowombat then installed `Cython>=3.x`, and the build-versus-package Cython conflicted.

Note that the above is a temporary solution. We will probably pin Cython in #247. If you want to test the PR branch prior to its release and provide feedback, you can install (clone branch of #247 for command-line parsing reasons):
```commandline
pip install git+https://github.com/jgrss/geowombat.git@issue_246
```
But we'll probably have #247 merged soon.

**Comment by mmann1123** [comment]
Also `mamba` is way better than `conda`... worth installing.

**Comment by jgrss** [comment]
This should be fixed with #247 and release v2.1.6, but please re-open if you still have issues.
